tldr/cross_file_calls/scanner.py

```
@@ -0,0 +1,306 @@
+            # Get appropriate parser for this file
+            parser = get_parser_for_file(file_path)
+            if not parser:
+                continue
+            
+            # Extract calls from file
+            file_calls = parser.extract_calls(file_path, timeout=file_timeout)
+            
+            # Add to call graph
+            call_graph.add_file(file_path, file_calls)
+            
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)